Makefile

```diff
@@ -245,9 +245,8 @@ cmd/migrate/migrate:
 #############
 # Releasing #
 #############
-# concurrency is limited to 2 to prevent CircleCI from OOMing. Sorry
-GOX = gox $(GO_FLAGS) -parallel=2 -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
-CGO_GOX = gox $(DYN_GO_FLAGS) -cgo -parallel=2 -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
+GOX = gox $(GO_FLAGS) -parallel=4 -output="dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
```

Comment:
curious why 4? (I mean can it be higher?)

Reply:
I removed it. We don't run the risk of OOMing in Drone, so I'll just let `gox` figure it out